javagen/src/main/java/com/azure/autorest/model/clientmodel/ProxyMethodExample.java

```diff
@@ -77,13 +73,9 @@ public Object getUnescapedQueryValue() {
         @Override
         public String toString() {
             try {
-                return "ParameterValue{" +
-                        "objectValue=" + PRETTY_PRINTER.writeValueAsString(objectValue) +
-                        '}';
+                return "ParameterValue{" + "objectValue=" + NORMAL_PRINTER.writeValueAsString(objectValue) + '}';
```

Comment:
I assume the change of PRETTY_PRINTER to NORMAL_PRINTER here is just because of this `toString` is only for logging purpose and we don't want to waste CPU here?

Reply:
That and eventually the hope is to replace with azure-json instead of Jackson to go full circle on stream-style.